src/main/java/io/github/opencubicchunks/cubicchunks/core/network/WorldEncoder.java

```diff
@@ -158,6 +167,16 @@ static void decodeCube(PacketBuffer in, List<Cube> cubes) {
                 cube.getStorage().recalculateRefCounts();
             }
         }
+        
+        // 6. biomes
+        for (int i = 0; i < cubes.size(); i++) {
+            if (!hasCustomBiomeMap[i])
+                continue;
+            Cube cube = cubes.get(i);
+            byte[] blockBiomeArray = new byte[8*8];
```

Comment:
Magic constants

Reply:
8=16>>1
What you mean?

Reply:
Yes I know where they come from. But it's far from obvious when someone doesn't know how it all works, and you should put it as a `public static final int` field somewhere (Coords class).